oer-bookr/src/components/book/Book.js

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import { Link } from "react-router-dom"
+
+import { BookWrapper, BookInfo } from "../../styles/bookStyles"
+
+import ancientCivilizations from "../../assets/images/categories/ancientCivilizations.jpg"
+import arts from "../../assets/images/categories/arts.jpg"
+import business from "../../assets/images/categories/business.jpg"
+import education from "../../assets/images/categories/education.jpg"
+import engineering from "../../assets/images/categories/engineering.jpg"
+import language from "../../assets/images/categories/language.jpg"
+import mathematics from "../../assets/images/categories/mathematics.jpg"
+
+const Book = props => {
+  let image = ""
+  switch (props.book.category) {
+    case "Ancient Civilizations":
+      image = ancientCivilizations
+      break
+    case "Mathematics":
+      image = mathematics
+      break
+    case "Arts and Humanities":
+      image = arts
+      break
+    case "Education":
+      image = education
+      break
+    case "Engineering":
+      image = engineering
+      break
+    case "Business and Communications":
+      image = business
+      break
+    case "Language":
+      image = language
+      break
+    default:
+      image = ""
+      break
+  }
+  return (
+    <BookWrapper>
+      <img src={image} alt="Book" />
+      <BookInfo>
+        <div>
+          <span>Title:</span>
+          <Link to={`/book/${props.book.id}`}> {props.book.title}</Link>
+        </div>
+        <div>
+          <span>Author:</span> {props.book.author}
+        </div>
+        <div>
+          <span>Publisher:</span> {props.book.publisher}
+        </div>
+        <div>
+          <span>License:</span> {props.book.license}
+        </div>
+        <div>
+          <span>Reviews:</span> {props.book.reviews}
+        </div>
```

Comment:
It feels like some of this code could be more programmatic; for example:
```javascript
// ...
const capitalize = str => str[0].toUpperCase() + str.slice(1)
const sections = [ 'author', 'publisher', 'license', 'reviews' ]
const makeSectionDiv = name => (
  <div>
    <span>{capitalize(section)}:</span> {props.book[section]}
  </div>
)

return  (
  <BookWrapper>
    <img /* ... */ />
    <BookInfo>
      <div>
        <span>Title:</span>
        <Link to={`/book/${props.book.id}`}> {props.book.title}</Link>
      </div>
      {sections.map(makeSectionDiv)}
    </BookInfo>
  </BookWrapper>
);
// ...
```
It also feels like the switch statement could be made simpler somehow in a similar sense. I don't have an immediate way of doing it coming to my head, though! I think it'd involve an object containing the images, though.

Reply:
Good idea for showing the sections of the book info. For the switch statement, it's just dummy data for now. I will be getting the information/images from backend (I think)

Reply:
Not sure how to display images otherwise. In React you have to import the images unless you have a url

Reply:
Yeah, it'd be importing and then storing them in an object after the fact, which would be adding an extra step in a sense, and then you could access them via their keys (the same as the variable names that you imported them as). 

You will be getting data from the back-end later, so I definitely agree there!